### PR TITLE
New home screen concept

### PR DESCRIPTION
**Pull request by carlostojal**
Being the home screen the first impression that the user has of the platform, I considered that it should be more captivating for both new and frequent users. The thumbnails shown are those of up to 150 most popular uploads. In this way, the initial screen would be a kind of incentive to explore the rest of the website's content.

Short demo here:
https://newtube.app/user/carlostojal/jzuB5Ff

### CODE REVIEW

**Comment by mayeaux** [comment]
Very cool! I will test this on mobile but otherwise I think I can merge that right away, nice work!

**Comment by mayeaux** [comment]
Is there a way to be 'zoomed out' of the sphere after the first animation? So you're outside of it? Right now when it starts you begin in the centre of the sphere and the uploads are "inside out"

**Comment by carlostojal** [comment]
Yes, it's possible.

File: views/public/home.pug [hunk 1]
```diff
@@ -4,97 +4,196 @@ block content
...
+    var targets = {helix: [], grid: [], sphere: []};
 
-      .row .col-sm-4 {
-        margin-bottom:52px;
-      }
+    var x = 0;
+
+    init();
+    animate();
+    // change(); // the thumbnails layout will changing every 10 seconds
```
Comment (carlostojal) [inline views/public/home.pug:59]: This is not recommended in instances with little or none content, because it will be a bit strange.

**Comment by carlostojal** [comment]
Done :)

**Comment by chovy** [comment]
I'm sure you put some work into it but please provide an env variable to disable this. I don't want it.

**Comment by carlostojal** [comment]
@chovy changed it now. Just set `HOME_SPHERE_ENABLED` to `false` in `.env.settings` to get the old home screen.

**Comment by mayeaux** [comment]
Let's do it this way: 

Let's have a route for app.get(/globe)

That will render that globe/sphere as a pug file 

Then we will leave home as it is , if people want to use the globe instead of the default landing home page they can change the res.render on the app.get('/') route to render that template instead of the default one.

I'd probably add an env var for 'default landing page' type of thing but it would require extra work because of how the popular section works atm.

**Comment by mayeaux** [comment]
Looks good. I'm gonna check this out and maybe change a couple things but will try and get it merged and deployed to NewTube for when you wake up tomorrow lol nice work

**Comment by mayeaux** [comment]
I'm gonna merge this, I think it can still use a bit of polish, it's a bit weird that the text is off to the left, I think it would be better if we could center the text and put the globe beneath it, but overall the concept is very very cool so I will merge it and deploy it to NewTube and maybe we can put some finishing touches on it in the next few days 
Screen Shot 2020-04-15 at 9 34 20 PM

**Pull request merged**